eng/Versions.props

```diff
@@ -195,6 +195,11 @@
     <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-7.0.100-bootstrap.11</PrivateSourceBuiltArtifactsPackageVersion>
     <PrivateSourceBuiltPrebuiltsPackageVersion>0.1.0-7.0.100-5</PrivateSourceBuiltPrebuiltsPackageVersion>
   </PropertyGroup>
+  <!-- dependencies for source-build VMR -->
```

Comment:
Nit - grouping dependencies by the depending code does not withstand the test of time.  By that I mean as new features are added across the repo, they may have these same dependencies.  if this happens the grouping like this no longer make sense.  Towards the top of this file, you will see the groupings are by source-repo.  This is more robust IMO.    I will note, there are a lot of exception to this generalization in this file...

Reply:
Ah, sure, I copied the example for tarball from above. I will throw them into their repo sections